libs/features/mas/src/inline-price.js

```diff
@@ -260,11 +260,13 @@ export class InlinePrice extends HTMLSpanElement {
         const version = this.masElement.togglePending(options);
         this.innerHTML = '';
         const [promise] = service.resolveOfferSelectors(options);
-        return this.renderOffers(
+        const result = this.renderOffers(
             selectOffers(await promise, options),
             options,
             version,
         );
+        this.masElement.notify();
```

Comment:
this logic is handled at mas-element level(*), the custom elements (we may have more in the future) should not know how / when to notify. 

(*) https://github.com/adobecom/milo/pull/3388/files#diff-b76179f008b9c485066e722d9e2b9265ccdbfe9d5abb33507d5ab8424e7a45b5L207-L219

Reply:
got it, i will move it back into mas-element